people-counting/people_counting/api/people_counter.py

```diff
@@ -58,9 +58,12 @@ def count_people_and_make_video(
     video_to_count_local_path = os.path.basename(
         input_of_people_counter.data.storage_path
     )
-    bucket_of_videos_to_count.blob(
-        input_of_people_counter.data.storage_path
-    ).download_to_filename(video_to_count_local_path)
+    blob_name = "/".join(
+        input_of_people_counter.data.storage_path.lstrip("gs://").split("/")[1:]
+    )
+    bucket_of_videos_to_count.blob(blob_name).download_to_filename(
+        video_to_count_local_path
+    )
```

Comment:
Obligé de faire ce changement car auparavant` input_of_people_counter.data.storage_path` était de la forme "`4dc4e2b58bdd700ed708317c1b24bf2cc518.avi`" et maintenant "`gs://pc-videos-to-count-pc-add-integration-tests/4dc4e2b58bdd700ed708317c1b24bf2cc518.avi`"